### DISCUSSION TITLE
Print a message if a newer version is available

### DISCUSSION THREAD

**Discussion by beardgans** [rome/tools#3132] [Suggestions]
Hi everyone!
Thank you for this awesome project!

I found this feature in the [GitHub CLI application](https://github.com/cli/cli/blob/trunk/internal/update/update.go) and [npm](https://github.com/npm/cli/blob/latest/lib/utils/update-notifier.js).

I think it'll be useful for users who install `rome/tools` using the installation script or `npm`.
And this will let them know that a new version is available and it's time to update.

Ready-made crates to implement that:
* https://github.com/mgrachev/update-informer
* https://github.com/tarikeshaq/update-notifier

**Comment by ematipico** [comment]
This is something that we will probably do once Rome will start being released in the `latest` tag on npm. At the moment we use the `next` tag, so it doesn't make much sense to have it.

**Comment by Conaclos** [comment]
I am not found of this kind of feature. This requires a network request.
